Monika After Story/game/script-holidays.rpy

```diff
@@ -2350,6 +2364,63 @@ label mas_d25_season_exit:
         mas_d25ReactToGifts()
     return
 
+# pre-d25 gift handling
+# NOTE: YES this is super hackish, but until we have event/topic context 
+#   this is the quickest way without adding an extra per var.
+# multiple d25, no non-d25
+label mas_d25_gift_tree_intro_2_0:
+    $ d25_multiple = True
+    $ nd25_any = False
```

Comment:
If this helps, those could be label arguments. You could also use `**kwargs`
```renpy
label mas_d25_gift_tree_intro(d25_multiple, nd25_any, **kwargs):
    # ...
    return
```
The only downside you'd have to use `call` instead of `jump`, but it doesn't look like it'd be a problem in this case.